overrides/main.html

```diff
@@ -33,5 +33,5 @@
   <meta name="twitter:image" content="https://dietpi.com/images/dietpi-logo_720x720.png">
 {% endblock %}
 {% block announce %}
-  <a href="https://dietpi.com/contribute.html">Learn how to support and contribute to the DietPi project <span class="twemoji">{% include ".icons/fontawesome/regular/heart.svg" %}</span></a>
+  <a href="https://dietpi.com">WEBSITE</a>&emsp;<a href="https://dietpi.com/#features">FEATURES</a>&emsp;<a href="https://dietpi.com/#download">DOWNLOAD</a>&emsp;<a href="https://dietpi.com/forum/">FORUM</a>&emsp;<a href="https://dietpi.com/blog/">BLOG</a>&emsp;<a href="https://dietpi.com/#gettingstarted">GETTING STARTED</a>&emsp;<a href="https://dietpi.com/contribute.html">DONATE</a>
```

Comment:
What about changing "WEBSITE" to "DIETPI HOME"?
May fit better to the "HOME" button on dietpi.com

Reply:
I had the same idea. Or even just "HOME". With "DOCS", "FORUM" etc it should be clear that it is the "home"page.